lib/ASRouterTargeting.jsm

```diff
@@ -25,14 +26,21 @@ const TargetingGetters = {
   },
   get hasFxAccount() {
     return Services.prefs.prefHasUserValue(FXA_USERNAME_PREF);
+  },
+  get isInExperimentCohort() {
```

Comment:
can you add a comment that this is a temporary targeting value for the purpose of the experiment?